NPE on ICP for 0.6.4-rc2

Traceback:
```
Exception in thread "main" java.lang.ExceptionInInitializerError
    at java.base/java.lang.J9VMInternals.ensureError(Unknown Source)
    at java.base/java.lang.J9VMInternals.recordInitializationFailure(Unknown Source)
    at com.kruize.collection.CollectMetrics.<init>(CollectMetrics.java:40)
    at com.kruize.main.Kruize.main(Kruize.java:108)
Caused by: java.lang.NullPointerException
    at com.kruize.environment.kubernetes.KubernetesEnvImpl.setMonitoringLabels(KubernetesEnvImpl.java:288)
    at com.kruize.environment.kubernetes.KubernetesEnvImpl.setupMonitoringAgent(KubernetesEnvImpl.java:65)
    at com.kruize.environment.EnvTypeImpl.getInstance(EnvTypeImpl.java:50)
    at com.kruize.environment.EnvTypeImpl.<clinit>(EnvTypeImpl.java:39)
```

--- a/src/main/java/com/kruize/environment/kubernetes/KubernetesEnvImpl.java
+++ b/src/main/java/com/kruize/environment/kubernetes/KubernetesEnvImpl.java
@@ -280,21 +280,18 @@ public class KubernetesEnvImpl extends EnvTypeImpl
 
     private void setMonitoringLabels()
     {
+        PrometheusQuery prometheusQuery = PrometheusQuery.getInstance();
+
         try {
             URL labelURL = new URL(DeploymentInfo.getMonitoringAgentEndpoint() + "/api/v1/labels");
             String result = HttpUtil.getDataFromURL(labelURL);
-            PrometheusQuery prometheusQuery = PrometheusQuery.getInstance();
-
+            
             if (result.contains("\"pod\"")) {
                 prometheusQuery.setPodLabel("pod");
                 prometheusQuery.setContainerLabel("container");
-            } else {
-                prometheusQuery.setPodLabel("pod_name");
-                prometheusQuery.setContainerLabel("container_name");
             }
-
-        } catch (MalformedURLException e) {
-            e.printStackTrace();
         }
+        /* Use the default labels */
+        catch (MalformedURLException | NullPointerException ignored) { }
     }
 }

--- a/src/main/java/com/kruize/query/PrometheusQuery.java
+++ b/src/main/java/com/kruize/query/PrometheusQuery.java
@@ -23,8 +23,8 @@ import com.kruize.query.kubernetes.KubernetesPrometheusQuery;
 public abstract class PrometheusQuery implements Query
 {
     private static PrometheusQuery prometheusQuery = null;
-    protected String podLabel;
-    protected String containerLabel;
+    protected String podLabel = "pod_name";
+    protected String containerLabel = "container_name";
 
     static {
         getInstance();

